service/src/test/java/greencity/service/UBSClientServiceImplTest.java

```diff
@@ -1108,6 +1109,57 @@ void saveFullOrderToDBForIF() throws IllegalAccessException {
         assertNotNull(result);
     }
 
+    @Test
+    void saveFullOrderToDBForIfTrue() throws IllegalAccessException {
```

Comment:
Could you rename this test, please? Because from context it is not clear what you are testing